aderyn_nodejs/src/lib.rs

```diff
@@ -0,0 +1,78 @@
+use aderyn_driver::{
+    detector::get_issue_detector_by_name,
+    driver::{self, Args},
+};
+use neon::prelude::*;
+
+fn drive(mut cx: FunctionContext) -> JsResult<JsBoolean> {
+    let root = cx.argument::<JsString>(0)?;
```

Comment:
order of arguments for `drive`

Reply:
Could we add some TS tests to this that run in CI to make sure reports are produced correctly @TilakMaddy ?